bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -87,6 +97,25 @@ spec:
           name: var-lib-openvswitch
         - mountPath: /env
           name: env-overrides
+        - mountPath: /run/systemd
```

Comment:
It may be simpler to mount the whole host, and use `chroot /host` for selected commands.  The MCD today just runs entirely chrooted into the host.